content/doc/book/system-administration/security/configuring-content-security-policy.adoc

```diff
@@ -0,0 +1,228 @@
+---
+title: Access Control for Builds
+layout: documentation
+---
+
+Jenkins 1.641 / Jenkins 1.625.3 introduce the
+`+Content-Security-Policy+` header to static files served by Jenkins
+(specifically `+DirectoryBrowserSupport+`). This header is set to a very
+restrictive default set of permissions to protect Jenkins users from
+malicious HTML/JS files in workspaces, `+/userContent+`, or archived
+artifacts.
+
+Unfortunately, several popular, useful plugins are affected by this and
+lose part of their functionality unless the default rules are relaxed.
+
+Alternatives
+
+TIP: Since Jenkins 2.200, it is possible to define a *Resource Root URL* in
+the Jenkins system configuration as an alternative to relaxing the
+Content Security Policy rules. See its inline help for details.
+
+
+
+[[ConfiguringContentSecurityPolicy-TheDefaultRuleSet]]
```

Comment:
Wiki generated anchors make no sense on the site. Better to just remove all of these.